lib/github_service/commands/cross_repo_test.rb

```diff
@@ -274,15 +274,15 @@ def update_github_workflow_yaml_content
         raw_yaml = rugged_repo.blob_at(branch_ref.target.oid, ".github/workflows/ci.yaml").content
         content  = YAML.safe_load(raw_yaml)
 
-        content["jobs"] = test_repos.each_with_object({}) do |test_repo, result|
-          result[test_repo] = {
+        content["jobs"] = {
+          "cross-repo" => {
             "uses" => "ManageIQ/manageiq-cross_repo/.github/workflows/manageiq_cross_repo.yaml@master",
             "with" => {
-              "test-repo" => test_repo,
+              "test-repo" => "[#{test_repos.map { |repo| "\"#{repo}\"" }.join(", ")}]",
```

Comment:
```suggestion
              "test-repo" => test_repos.inspect,
```

Ruby array syntax is JSON compliant, so this is a lot simpler.

Reply:
Ah thanks much better, felt like there had to be a better way to do that :laughing: